app/cdap/components/ConnectionList/Components/TabLabelCanSample/__test__/TabLabelCanSample.test.tsx

```diff
@@ -20,37 +20,43 @@ import TabLabelCanSample from '../index';
 import { mockConnectorTypeData, mockEntityDataForNoWorkspace } from '../mock/mockConnectorTypeData';
 import { Route, Router, Switch } from 'react-router';
 import * as apiHelpers from 'components/Connections/Browser/GenericBrowser/apiHelpers';
-import { createBrowserHistory } from 'history';
-
-const history = createBrowserHistory({
-  basename: '/',
-});
+import history from 'services/history';
 
 describe('Test TabLabelCanSample Component', () => {
   it('Should render TabLabelCanSample Component', () => {
     render(
-      <TabLabelCanSample
-        label={mockConnectorTypeData.name}
-        entity={mockConnectorTypeData}
-        initialConnectionId={undefined}
-        toggleLoader={() => null}
-        setIsErrorOnNoWorkSpace={jest.fn()}
-      />
+      <Router history={history}>
+        <Route>
+          <TabLabelCanSample
+            label={mockConnectorTypeData.name}
+            entity={mockConnectorTypeData}
+            initialConnectionId={undefined}
+            toggleLoader={() => null}
+            setIsErrorOnNoWorkSpace={jest.fn()}
+            dataTestId={'connections-tab-ref-label-simple'}
```

Comment:
this should be data-testid

Reply:
this shows up in a couple of places and should be data-testid for attributes

Reply:
*since they're html attributes and not props

Reply:
hi sean, we are sending the dataTestId from parent to child in props as we need to have unique names
but in test files it seems like a constant label

so, usecase wise this is correct implementation

thank you